Article/Article.js

```diff
@@ -26,4 +35,5 @@ class Article {
 
 */
 
-let articles;
\ No newline at end of file
+let articles = [];
+document.querySelectorAll(".article").forEach(el =>  articles.push(new Article(el)));
```

Comment:
you could also write this chain of method calls directly into the binding for articles without using push >>

```
let articles = document
  .querySelectorAll('.article')
  .forEach(el => new Article(el));
```